sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -765,7 +765,7 @@ object SQLConf {
         "feature flag.")
       .version("4.0.0")
       .booleanConf
-      .createWithDefault(Utils.isTesting)
+      .createWithDefault(true)
```

Comment:
can we grep `spark.sql.collation.enable` in the codebase? IIRC we explicitly enable this conf in some examples of SQL function doc.